R/git.R

```diff
@@ -0,0 +1,22 @@
+#' Git information
+#' 
+#' Displays git information from the repository
+#' 
+#' @import git2r
+#' @import shiny
+#' @export
+git_info <- function() {
+  print(getwd())
+  repo <- init(path = ".", bare = FALSE)
+  branch <- repository_head(repo)[[1]]
+  if (is.null(branch)) {
+    git_message <- "Empty repo."
+  } else {
+    last_commit <- as.character(commits(repo)[[1]])[[4]]
+    git_message <- span(paste("Branch:", branch),
```

Comment:
I suggest using `glue()` for concatenating text. You could easier notice that you added different number of spaces.

```
glue("Branch: {branch}{br()}Last commit: {last_commit}")

Reply:
You're right. I used `glue` with some html styling so it looks as in the fig below.